app/lib/Plugins/SearchEngine/Elastic8.php

```diff
@@ -515,72 +540,80 @@ public function commitRowIndexing() {
 	 * @param int|null $field_tablenum
 	 * @param int|null|array $field_nums
 	 * @param int|null $field_row_id
+	 * @param int|null $rel_type_id
+	 *
+	 * @throws ApplicationException
+	 * @throws AuthenticationException
+	 * @throws ClientResponseException
+	 * @throws MissingParameterException
+	 * @throws ServerResponseException
+	 * @throws Exception
 	 */
 	public function removeRowIndexing(
 		int $subject_tablenum, int $subject_row_id, ?int $field_tablenum = null, $field_nums = null,
```

Comment:
```suggestion
		int $subject_tablenum, int $subject_row_id, ?int $field_tablenum = null, ?array $field_nums = null,
```

Reply:
this is currently `int|null|array` in the PHPDoc comment, happy to go with your suggestion and we can switch it back if we find it is needed!

Reply:
Updating the signature is incompatible with the base class, have reverted this change, but updated the PHPDoc tag.